### ISSUE TITLE
Nightly GitHub Action failure

### ISSUE THREAD

**Issue by robert-scheck** [openbgpd-portable/openbgpd-portable#38]
The nightly GitHub Action for the container image [fails](https://github.com/openbgpd-portable/openbgpd-container/runs/7823807351?check_suite_focus=true#step:9:691) for quite some time with:

```
#34 28.71 Copying bgplgd.h to bgplgd
#34 28.71 Copying http.h to bgplgd
#34 28.72 Copying slowcgi.h to bgplgd
#34 28.72 Patching patches/0001-depend-on-carp-requieres-HAVE_CARP.patch
#34 28.73 Patching patches/0002-set-IPV6_V6ONLY-if-we-are-binding-to-an-IPv6-address.patch
#34 28.74 Patching patches/0003-Special-handle-IPV6_MINHOPCOUNT-and-IP_MINTTL-since-.patch
#34 28.78 Patching patches/0004-Allow-overriding-default-user-and-file-locations.patch
#34 28.78 Patching patches/0005-Add-placeholders-to-manual-pages.patch
#34 28.98 Patching patches/0006-Add-placeholders-and-compat-fixes-to-bgplgd.patch
#34 28.98 1 out of 3 hunks FAILED -- saving rejects to file bgplgd/slowcgi.c.rej
```

**Comment by cjeker** [comment]
Fixed by 697aa52d723f3840f9c71dcea30688c1d85a526b

**Comment by robert-scheck** [comment]
Yes and no. Now this [fails](https://github.com/openbgpd-portable/openbgpd-container/runs/7879842023?check_suite_focus=true#step:9:2000) with:

```
#34 123.4 cc -DPACKAGE_NAME=\"OpenBGPD\" -DPACKAGE_TARNAME=\"openbgpd\" -DPACKAGE_VERSION=\"7.5\" -DPACKAGE_STRING=\"OpenBGPD\ 7.5\" -DPACKAGE_BUGREPORT=\"\" -DPACKAGE_URL=\"\" -DPACKAGE=\"openbgpd\" -DVERSION=\"7.5\" -DSPT_TYPE=SPT_REUSEARGV -DHAVE_STDIO_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_STRINGS_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_UNISTD_H=1 -DSTDC_HEADERS=1 -DHAVE_DLFCN_H=1 -DLT_OBJDIR=\".libs/\" -DHAVE_ENDIAN_H=1 -DHAVE_LINUX_IN6_H=1 -DHAVE_CLOCK_GETTIME=1 -DHAVE_ASPRINTF=1 -DHAVE_EXPLICIT_BZERO=1 -DHAVE_GETENTROPY=1 -DHAVE_REALLOCARRAY=1 -DHAVE_SETGROUPS=1 -DHAVE_SETREGID=1 -DHAVE_SETRESGID=1 -DHAVE_SETREUID=1 -DHAVE_SETRESUID=1 -DHAVE_STRLCAT=1 -DHAVE_STRLCPY=1 -DHAVE_SYSCONF=1 -DHAVE_LIBMNL=1 -DHAVE_MNL_SOCKET_RECVFROM=1 -DHAVE_ERR_H=1 -DHAVE_EVENT_H=1 -DHAVE_LIBEVENT=1 -DHAVE_EVENT_INIT=1 -DHAVE_EVENT_DISPATCH=1 -DBGPD_USER=\"bgpd\" -DBGPLGD_USER=\"_bgplgd\" -I.  -I../../include -I../../src/bgpd  -D_DEFAULT_SOURCE -D_BSD_SOURCE -D_GNU_SOURCE -Wall -Wstrict-prototypes -Wmissing-prototypes -Wmissing-declarations -Wshadow -Wpointer-arith -Wcast-qual -Wsign-compare -Wno-pointer-sign  -DSYSCONFDIR=\"/etc/bgpd\" -DRUNSTATEDIR=\"/run/bgpd\" -g -O2  -MT bgpd-kroute-linux.o -MD -MP -MF .deps/bgpd-kroute-linux.Tpo -c -o bgpd-kroute-linux.o `test -f 'kroute-linux.c' || echo './'`kroute-linux.c
#34 123.9 kroute-linux.c: In function 'kroute_match':
#34 123.9 kroute-linux.c:453:41: warning: implicit declaration of function 'bgpd_filternexthop'; did you mean 'bgpd_oknexthop'? [-Wimplicit-function-declaration]
#34 123.9   453 |                         if (matchall || bgpd_filternexthop(kr_tofull(kr)) == 0)
#34 123.9       |                                         ^~~~~~~~~~~~~~~~~~
#34 123.9       |                                         bgpd_oknexthop
#34 123.9 kroute-linux.c: In function 'dispatch_rtmsg_addr':
#34 123.9 kroute-linux.c:924:21: error: 'F_KERNEL' undeclared (first use in this function)
#34 123.9   924 |         kl->flags = F_KERNEL;
#34 123.9       |                     ^~~~~~~~
#34 123.9 kroute-linux.c:924:21: note: each undeclared identifier is reported only once for each function it appears in
#34 124.0 kroute-linux.c: In function 'kroute_insert':
#34 124.0 kroute-linux.c:1771:25: error: 'F_KERNEL' undeclared (first use in this function)
#34 124.0  1771 |         if (kr->flags & F_KERNEL) {
#34 124.0       |                         ^~~~~~~~
#34 124.0 kroute-linux.c: In function 'kroute_remove':
#34 124.0 kroute-linux.c:1841:25: error: 'F_KERNEL' undeclared (first use in this function)
#34 124.0  1841 |         if (kr->flags & F_KERNEL && kr == krm && kr->next == NULL)
#34 124.0       |                         ^~~~~~~~
#34 124.0 kroute-linux.c: In function 'kroute6_insert':
#34 124.0 kroute-linux.c:1927:25: error: 'F_KERNEL' undeclared (first use in this function)
#34 124.0  1927 |         if (kr->flags & F_KERNEL) {
#34 124.0       |                         ^~~~~~~~
#34 124.0 kroute-linux.c: In function 'kroute6_remove':
#34 124.0 kroute-linux.c:1998:25: error: 'F_KERNEL' undeclared (first use in this function)
#34 124.0  1998 |         if (kr->flags & F_KERNEL && kr == krm && kr->next == NULL)
#34 124.0       |                         ^~~~~~~~
#34 124.0 kroute-linux.c: In function 'kr_redistribute':
#34 124.1 kroute-linux.c:2044:27: error: 'F_KERNEL' undeclared (first use in this function)
#34 124.1  2044 |         if (!(kr->flags & F_KERNEL))
#34 124.1       |                           ^~~~~~~~
#34 124.1 kroute-linux.c: In function 'kr_redistribute6':
#34 124.1 kroute-linux.c:2091:28: error: 'F_KERNEL' undeclared (first use in this function)
#34 124.1  2091 |         if (!(kr6->flags & F_KERNEL))
#34 124.1       |                            ^~~~~~~~
#34 124.1 kroute-linux.c: In function 'kif_kr_insert':
#34 124.1 kroute-linux.c:2262:31: error: 'F_DOWN' undeclared (first use in this function)
#34 124.1  2262 |                 kr->flags &= ~F_DOWN;
#34 124.1       |                               ^~~~~~
#34 124.1 kroute-linux.c: In function 'kif_kr6_insert':
#34 124.1 kroute-linux.c:2321:31: error: 'F_DOWN' undeclared (first use in this function)
#34 124.1  2321 |                 kr->flags &= ~F_DOWN;
#34 124.1       |                               ^~~~~~
#34 124.1 kroute-linux.c: In function 'kr_fib_delete':
#34 124.1 kroute-linux.c:2377:35: error: 'F_KERNEL' undeclared (first use in this function)
#34 124.1  2377 |                 if (!(kr->flags & F_KERNEL))
#34 124.1       |                                   ^~~~~~~~
#34 124.3 make[1]: *** [Makefile:795: bgpd-kroute-linux.o] Error 1
#34 124.3 make[1]: Leaving directory '/tmp/openbgpd-portable/src/bgpd'
#34 124.3 make: *** [Makefile:403: all-recursive] Error 1
```
